### PR TITLE
We're unofficial (so let's tell the world so)

### PR DESCRIPTION
**Pull request by hawkrives**
Partially tongue-in-cheek, but I do think that we need something like this.

Wording of messages, layout, and behavior (currently there is none) is all open to discussion.

### CODE REVIEW

**Comment by drewvolz** [comment]
As always, I’ll be the Debbie Downer. I’d like to request the wording to change.

**Comment by hawkrives** [comment]
That's perfectly fine, but I'm going to come back at'cha with a request for suggestions ;-)

**Comment by drewvolz** [comment]
- An unofficial St. Olaf app
- For students, by students
- For Oles, by Oles
- Not supported by St. Olaf

**Review by hannesmcman — DISMISSED** [review]

File: source/views/home/notice.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import sample from 'lodash/sample'
+
+const messages = [
+	'☃️ An Unofficial App Project ☃️',
+	'For students, by students',
+	'By students, for students',
+	'SGA-owned and misunderstood',
+	'I AM AN UNOFFICIAL APP',
+	'we are (not) olaf',
+	'☃️',
```
Comment (drewvolz) [inline source/views/home/notice.js]: Do we really want this

File: source/views/home/notice.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+import * as c from '../components/colors'
+import sample from 'lodash/sample'
+
+const messages = [
+	'☃️ An Unofficial App Project ☃️',
+	'For students, by students',
+	'By students, for students',
+	'SGA-owned and misunderstood',
+	'I AM AN UNOFFICIAL APP',
+	'we are (not) olaf',
```
Comment (drewvolz) [inline source/views/home/notice.js]: This one doesn't tell me much

File: source/views/home/notice.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+import * as React from 'react'
+import * as glamorous from 'glamorous-native'
+import * as c from '../components/colors'
+import sample from 'lodash/sample'
+
+const messages = [
+	'☃️ An Unofficial App Project ☃️',
+	'For students, by students',
+	'By students, for students',
+	'SGA-owned and misunderstood',
```
Comment (drewvolz) [inline source/views/home/notice.js]: Not necessarily true

File: source/views/home/notice.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+	'I AM AN UNOFFICIAL APP',
```
Comment (drewvolz) [inline source/views/home/notice.js]: Not a fan of the use of "I" here

**Review by drewvolz — CHANGES_REQUESTED** [review]

**Comment by drewvolz** [comment]
What we say on the front of the app is very important to me. This is what is forward facing and what the user will be confronted with _every single time they use the app_, and what new users will first read. I'd rather be generic than confusing. I'd like the messages to be straightforward, which is why I suggested what I said in this message https://github.com/StoDevX/AAO-React-Native/pull/2419#issuecomment-372043402.

**Review by drewvolz — APPROVED** [review]
👍 :shipit:

**Pull request merged**